pre-docs/writing_plugins.md

```diff
@@ -0,0 +1,59 @@
+# Writing plugins
```

Comment:
This duplicates "plugins.md"

Reply:
Good catch. I've removed it from this branch.